release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -275,6 +275,18 @@ The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled
 [id="ocp-4-8-scale"]
 === Scale
 
+[id="ocp-4-8-scale-cluster-maximums"]
+==== Cluster maximums
+
+Updated guidance around xref:../scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc#planning-your-environment-according-to-object-maximums[cluster maximums] for {product-title} {product-version} is now available.
+
+[IMPORTANT]
+====
+No large scale testing for performance against Open Virtual Network (OVN) Kubernetes testing was executed for this release.
```

Comment:
@ahardin-rh, fyi, we're calling this OVN-Kubernetes throughout the networking docs, if this is about the network provider itself: https://docs.openshift.com/container-platform/4.7/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html

Reply:
@jboxman Thank you for pointing this out! I will update the wording to align.